07a_system_activity_plots.R

```diff
@@ -289,24 +290,33 @@ syso_detailBox <- reactive({
     getNameByValue(syso_project_types, input$syso_project_type),
     br(),
     strong("Age: "),
-    # if_else(
-    #   setequal(syso_age_cats, input$syso_age) |
-    #     is.null(input$syso_age),
-    #   "All Ages"#,
-    #   # getNameByValue(syso_age_cats, input$syso_age)
-    # ),
-    " | ",
+    if_else(length(input$syso_age) == 11,
+            "All Ages",
+            paste(input$syso_age, collapse = ', ')),
+    br(),
     strong("Gender: "),
-    # getNameByValue(syso_gender_cats(), input$syso_gender),
-    " | ",
+    if_else(length(input$syso_gender) == 5, 
+            # this works only because there are 5 categories in both
+            # inclusive and exclusive methodologies so this will need
+            # rewriting if the number of categories changes 
+            "All Genders",
+            getNameByValue(syso_gender_cats(), input$syso_gender)),
+    br(),
     strong("Race/Ethnicity: "),
-    # getNameByValue(syso_race_ethnicities, input$syso_race_ethnicity),
-    " | ",
+    str_sub(
+      getNameByValue(
+        unlist(syso_race_ethnicity_cats(input$methodology_type)),
+        input$syso_race_ethnicity),
+      start = str_locate(getNameByValue(
+        unlist(syso_race_ethnicity_cats(input$methodology_type)),
+        input$syso_race_ethnicity), "\\.")[,1] + 1,
+      end = -1L),
+    br(),
     strong("Special Populations: "),
-    # getNameByValue(syso_spec_pops_cats(), input$syso_spec_pops), 
+    getNameByValue(syso_spec_pops_people, input$syso_spec_pops),
```

Comment:
It looks like syso_spec_pops_cats() used to dynamically call syso_spec_pops_people or syso_spec_pops_hoh, depending on methodology type. But syso_spec_pops_hoh doesn't seem to be a thing. So I would just delete the commented out line here and the syso_spec_pops_cats() reactive in the server.R file.